utils.py

```
@@ -0,0 +1,646 @@
+from main import Logger
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)